web/packages/teleport/src/lib/tdp/client.ts

```diff
@@ -254,6 +255,9 @@ export default class Client extends EventEmitterWebAuthnSender {
         case MessageType.SHARED_DIRECTORY_LIST_REQUEST:
           this.handleSharedDirectoryListRequest(buffer);
           break;
+        case MessageType.SHARED_DIRECTORY_TRUNCATE_REQUEST:
+          this.handleSharedDirectoryTruncateRequest(buffer);
+          break;
         default:
           this.logger.warn(`received unsupported message type ${messageType}`);
```

Comment:
Looks like this is backwards compatible by default because an old client will hit this default case and log the message but otherwise continue to function. Is that right?

Reply:
Yeah, the truncate would simply be ignored were it to reach here. However, that would mean we have a vN w_d_s operating with a v(N-1) proxy, which afaik isn't supported in general (though in this case it is, incidentally).